src/main/java/leo/task/Task.java

```diff
@@ -0,0 +1,201 @@
+package leo.task;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import leo.ui.Ui;
+
+public class Task implements Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
Hey, I think this needs to be in all caps?

> A constant can be named using ALL_CAPS.

Reply:
Hi Francis, thanks for your comments. Pertaining to this field, the JavaDocs for implementing a `Serializable` object prescribes the field be named as such.

> The serialization runtime associates with each serializable class a version number, called a serialVersionUID, which is used during deserialization to verify that the sender and receiver of a serialized object have loaded classes for that object that are compatible with respect to serialization...:
 `ANY-ACCESS-MODIFIER static final long serialVersionUID = 42L;`